src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
>Could be just debugtool() then?

I guess. I just wrote 'command' to extend what appeared to be precedent from elsewhere. I just feel whatever it is needs to be precedented, and follow (or extend) any existing patterns.

>Or make it debuggerpath() and use it in conjunction with debugger()? So we get the tool's executable name from debugger() and combine it with debuggerpath()?

Nar, 'debugger' needs to be something high-level and abstract. 'Default' isn't an executable, and 'VisualGDB', 'WinGDB', 'Mago' are VS plugins. They don't have any such executable name, so I don't think that pattern works.

It also leads to debugger, debuggerpath... debuggerargs, and i'd say debuggerargs is too similar to debugargs. People would look at that and never perceive the distinction.
That's mainly why I went with 'debugtool', I think I felt something about the word 'tool' really made the distinction clear.